multiapps-common/src/main/java/org/cloudfoundry/multiapps/common/util/JsonUtil.java

```diff
@@ -39,98 +42,167 @@ private static ObjectMapper createObjectMapper() {
                                  .registerModule(new JavaTimeModule());
     }
 
+    private static ObjectMapper createObjectMapperWithNulls() {
+        return new ObjectMapper().setVisibility(PropertyAccessor.ALL, JsonAutoDetect.Visibility.NONE)
+                                 .setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY)
```

Comment:
Why visibility should be set?

Reply:
If they are not set Jackson fails to serialize/deserialize immutable values, only the fields should be used otherwise it throws NPEs

Reply:
In such case it will be better to add comment in the code why this is needed.

Reply:
Done